docs/plans/2026-05-08-feat-yellow-council-v2-four-cli-plan.md

```
@@ -0,0 +1,720 @@
+- `summary=` line — present?
+- `fenced_output_path=` line — present?
+- `findings_block_begin` / `findings_block_end` delimiters — present?
+
+Document the gap in the PR-0 description.
+
+### Task 0.2: Rewrite codex-reviewer Step 7 to emit the 6-key block (S, 2h)
+
+**File:** `plugins/yellow-codex/agents/review/codex-reviewer.md`
+
+Match gemini-reviewer's output exactly:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)